metadata/metadata_tutorial_addition.Rmd

```diff
@@ -0,0 +1,353 @@
+---
+title: "Metadata Upload Tutorial Part 1: Adding new metadata to BETYdb"
+author: "Kimberly Huynh"
+date: "4/30/2019"
+output: html_document
+---
+
+# Overview
+
+The objective of this tutorial is to demonstrate how to upload a season's metadata to the traits database. All traits data need to have associated metadata.
+
+Each season should have metadata for experiments, sites, treatments, cultivars, and citations.
+
+For this tutorial, you will be uploading your metadata to a test database. A tutorial on how to set up a local instance of a test database can be found in the [metadata folder of the terraref tutorials repository](https://github.com/terraref/tutorials/tree/test_db_tutorial/metadata).
+
+This tutorial will only cover how to add new experiments, sites, treatments, cultivars, and citations. 
+
+The `Metadata Upload Tutorial Part 2: Associating metadata in BETYdb` tutorial should be viewed after you have completed this tutorial and all new metadata have been added (see below for more details on how to run this markdown file). The second tutorial will cover how to associate the metadata that you add.
+
+# Required input
+
+URL to a public published google sheet (you will need to provide this url in the URL set up section)
+
+   * copy and fill out the following [google sheets template](https://docs.google.com/spreadsheets/d/1pNEuwNHNHHrpql0b36DYFAbOlBU5TJVF_nO4RdRzdmg/edit#gid=279516453).
+   * instructions on how to fill out the template can be found under the README worksheet.
+   * make the google sheet public (to anyone with link) and publish the entire document to the web as a xlsx.
+
+# How to Run this Rmd
+
+Knitting this markdown file will not upload any data to the test database. Being able to knit this file means that you will not encounter any errors when uploading your metadata. All experiments, sites, treatments, cutlivars, and citations can be uploaded successfully. 
+
+To actually commit additions to the test database, you will need to make an edit to this markdown file. 
+
+The value of `commit` needs to be changed from `FALSE` to `TRUE` in the following chunk:
+
+```{r commit-param}
+
+commit <- FALSE
+
+```
+
+It is important to note that you should only be uploading your metadata once. 
+
+Trying to upload your metadata more than once will result in errors. 
+
+If you would like to reinitialize the test database and remove any additions that you have made, re-run step 3 of the `How to set up local instance of BETY test database` tutorial.
+
+# Set up
+
+## Load packages
+
+The following packages will need to be loaded: `RPostgres`, `DBI`, `dplyr`, `googlesheets`, `kableExtra`, and `glue`. The `metadata` upload package will first need to be installed from github and then loaded in. 
+
+```{r load-pack, message = FALSE}
+
+if(!require(metadata)){
+  install_github('kimberlyh66/betydbtools@metadata_package', subdir = 'metadata')
```

Comment:
Once [the PR](https://github.com/az-digitalag/betydbtools/pull/2) for the version of the package in the betydbtools repo is accepted, it should be installed instead of this version that's in your local repo.